toolkit/imageconfigs/scripts/setup_cvm_image.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Should this code be added directly to the `imager` instead of as a script?

Reply:
This customization is quite specific to the CVM boot flow `UEFI -> shim -> sd-boot -> UKI`, and the setup is subject to change, so I think using this script is easier to maintain, at least for now.